### DISCUSSION TITLE
blog/building-a-wireless-corne-keyboard/

### DISCUSSION THREAD

**Discussion by giscus** [armno/armno.in.th#44] [Blog Post Comments]
# blog/building-a-wireless-corne-keyboard/

Upgrading my Corne split keyboard into a wireless version with Nice!nano.

https://armno.in.th/blog/building-a-wireless-corne-keyboard/

**Comment by mattsn0w** [comment]
How do you handle powering off the keyboard? I have a v3 board that I am curious about making wireless, but I want to be able to power it off.

**Reply by armno** [reply]
hey @mattsn0w 

For this keyboard I don't handle powering off actually. I just leave it on all the time. 

But I think you can put a switch to one of the wires to of the batteries like this https://www.youtube.com/watch?v=bfElyGM2B_k for more sophisticated option would be https://github.com/jhelvy/wireless-corne/tree/main/build#battery-toggle-switch.

or, build another board (😄) with a PCB that has a power switch slot on it. https://github.com/rafaeldelboni/buildlogs/blob/main/crkbd-choc-spaced-switch.md#power-switch. I went with this route actually.
